configserver/src/main/java/com/yahoo/vespa/config/server/filedistribution/FileDistributionFactory.java

```diff
@@ -17,15 +18,26 @@
 public class FileDistributionFactory {
 
     protected final ConfigserverConfig configserverConfig;
-    private final Supervisor supervisor = new Supervisor(new Transport("filedistribution"));
```

Comment:
Any reason why the supervisor/transport was moved ?
now you will have many instead of 1.
Independently it should have been closed.

Reply:
No, this should have been kept, will fix